### ISSUE TITLE
NIHMS schema should require an ISSN and publication type for the issns object array

### ISSUE THREAD

**Issue by emetsger** [OA-PASS/metadata-schemas#25]
Each item in the `issns` object array must have an issn and a publication type.  If publication type is missing with issn present, or issn is present with publication type missing, then the schema should fail validation.

**Comment by htpvu** [comment]
This does not align 100% with NIHMS requirements. ISSN and associated pubtype is ONLY required IF NLMTA is missing. 

So the requirement is submission must have NLMTA OR (ISSN AND PUBTYPE)

**Comment by birkland** [comment]
At present, it the nlmta schema requires:
```
        "prerequisites": {
            "anyOf": [
                {"$ref": "#/definitions/nlmta_present"},
                {"$ref": "#/definitions/issn_present"}
            ]
        }
```

which good for the NLMTA OR (ISSN) part, but:
```
        "issn_present": {
            "type": "object",
            "required": [
                "issns"
            ],
            "properties": {
                "issns": {
                    "$ref": "global.json#/properties/issns"
                }
            }
        },
```
This says that, failing the presence of an NLMTA, it's OK to have issns without pubType.  So this part needs to be updated to add pubType as a required issn property in order to fully capture NLMTA OR (ISSN AND PUBTYPE)

Note:  these constraints are present only in the nlmta schema (i.e. the NLMTA OR (ISSN AND PUBTYPE)
 rule applies only when the nihms schema is among the schemas apropos to the blob)

**Comment by birkland** [comment]
Closed by #28
